eck.md

```diff
@@ -128,19 +137,176 @@ spec:
             # we have to explicitly agree to patch the KBN binaries (the patching step will be done only once)
             - name: I_UNDERSTAND_IMPLICATION_OF_KBN_PATCHING
               value: "yes"
-            # we have to provide a ROR license if we want to use ROR Pro or Enterprise
+            # we have to provide a ROR license if we want to use ROR Pro or Enterprise (if the license is not provided, then ROR Free is used)
             - name: ROR_ACTIVATION_KEY
               value: "<YOUR_ACTIVATION_KEY/>"
 ```
 
 And these are all differences we need to make to run [ECK Quickstart](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html) with ReadonlyREST instead of X-Pack security. 
 
+## Using Docker images built by you and stored in your registry
+
+As you probably noticed, our docker images have to be run with root privileges. It's due to legal reasons and you, as a user, 
+have to confirm that you agree to do the patching steps. If running POD with root privileges is something you cannot accept, you can create your own image with the patching step done at the image creation level (not in the runtime as our image does) and place it your your registry.
+
+### Elasticsearch with ROR custom image
```

Comment:
this can be on its own page?

Reply:
The reason I wanted to isolate this special case into its own space is to keep the basic guide small and easy. People who want their own images bla bla it's something way more involved and I don't want people to get scared of it the first time they land here.

I guess the collapsible can do.

Reply:
ok, done. Please, take a look

Reply:
ok, done. please take a look

Reply:
@sscarduzio is it ok now?